### ISSUE TITLE
Drone Goes Crazy in Midair

### ISSUE THREAD

**Issue by ktlmoore** [eschnou/ardrone-autonomy#13]
Hi,

So I'm using the autonomy library to control my drone and most of the time things seem to go fine, however occasionally I will give it instructions and it will suddenly go crazy while staying in roughly the same position - it will buck around, rotate a bit and then eventually declare it did what it was told to do.

Specifically and most recently I tried the following instructions: [UP 0.32, FORWARD 0.8] which translate accordingly to the correct mission instructions.

Any advice?

**Comment by eschnou** [comment]
The issue may come from the drone attempting to correct both its x,y,z position and its yaw at the same time, which is usually not working great. Disabling the yaw control may help, but that requires diving in the code. Let me know you find a good tweak, happy to accept a pull request :-)

**Comment by ktlmoore** [comment]
It appears that it was to do with objects on the floor interfering with it, as it hasn't happened since I moved to a larger testing area.  There's a new issue though, but I'll move that to a new thread.  Thanks :)

**Comment by ktlmoore** [comment]
Nope, turns out that this isn't fixed.  It appears to happen without any particular pattern - sometimes it will take off and act normally, other times it will do this.  Whereabouts in the code is the yaw control likely to be?  I've been using the module but not developing it, so I don't know where I would really start looking, or what consequences would occur from disabling it

**Comment by ktlmoore** [comment]
So this is bizarre, it seems to only happen if the drone has not disconnected from the video feed...

**Comment by shaikh-shahid** [comment]
Any fix you guys recommend? I am facing the same issue.
